src/Extensions/Logging/Internal/EventSource/OmexLogEventSender.cs

```diff
@@ -41,7 +41,7 @@ public void LogMessage(Activity? activity, string category, LogLevel level, Even
 			string buildVersion = m_executionContext.BuildVersion;
 			string machineId = m_executionContext.MachineId;
 
-			string tagName = eventId.Name;
+			string tagName = eventId.Name ?? string.Empty;
```

Comment:
Can eventId be null here too?

Reply:
It's a struct, so we safe there

Reply:
as in should this be `eventId?.Name` and does the below code need to be updated to handle null eventId values also?

Reply:
`eventId` is of type [EventId](https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.logging.eventid?view=dotnet-plat-ext-3.1) that is readonly struct, so it could have `null` value, `default` value for it will be `EventId { Id: 0, Name: null }`.